coverage-tests.js

```diff
@@ -2162,6 +2162,30 @@ test('should return aborted tests in getAllTestResults', {
   },
 })
 
+test('Should not fail all by some aborted tests', {
+  // the test page has '@media query' to hide the 'region' element once width is less than 700
+  vg: true,
+  config: {
+    browsersInfo: [
+      { name: 'chrome', width: 1000, height: 400 }, // should succeed
+      { name: 'firefox', width: 680, height: 420 }, // should fail
+      { name: 'chrome', width: 500, height: 500 }, // should fail
+      { name: 'firefox', width: 1024, height: 768 }, // should succeed
+    ],
+  },
+  test({ driver, eyes, assert }) {
+    driver.visit('https://applitools.github.io/demo/TestPages/SimpleTestPage/mediaQuery.html')
+    eyes.open({ appName: 'Applitools Eyes SDK', viewportSize })
+    eyes.check({ region: '#size-dependent-div', isFully: true, fully: true })
+    assert.throws(() => void eyes.close())
+    const results = eyes.runner.getAllTestResults(false)
+    assert.equal(results.getAllResults()[0].testResults.isAborted, false)
+    assert.equal(results.getAllResults()[1]._container.exception.message.substring(0, 27), `failed to render screenshot`)
+    assert.equal(results.getAllResults()[2]._container.exception.message.substring(0, 27), `failed to render screenshot`)
```

Comment:
you don't do that. This way you just read internal javascript property, and we should work only through the TestResults interface. We also should do it in a language-agnostic way. You should always keep it in mind during working here

Reply:
I totally agree with that, my test was inspired by an existing test:
https://github.com/applitools/sdk.coverage.tests/blob/17c9f1fa89d955c863affe88f533ca8e1674b40b/coverage-tests.js#L2357
I will figure out what/how to upgrade it

Reply:
@kyryloonufriiev 
I removed the `._container` reference but left the message parsing method, except 'expecting' to have an 'exception' on two of the environments tests... what else I may verify? the failed tests have no 'TestResults' element inside `getAllResults()` they have only `exception` element (there are four environments, two should pass, two fail)